lms/templates/main.html

```diff
@@ -85,30 +92,7 @@
 
   <%block name="headextra"/>
 
-<%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
```

Comment:
I think that's a good idea. Since Stanford-style themes have to be modified to work with comprehensive theming anyway, it probably makes sense to go all the way and require to convert them to comprehensive-style themes.